html5/browser/api/clipboard.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+/**
+
+AUCTION:
+taskQueue
+Clipboard.setString()  NOW not works, facing to user-act lose of taskQueue.
+
+works in Chrome Firefox Opera. but not in Safari.
+@see https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand#Browser_compatibility
+
+Clipboard.getString() unimplemented. There is no easy way to do paste from clipboard to js variable.
+
+So look out your app behavior, when downgrade to html5 render.
+Any idea is welcome.
+**/
+
+const WEEX_CLIPBOARD_ID = '__weex_clipboard_id__'
+
+const clipboard = {
+
+  getString: function (callbackId) {
+    // not supported in html5
+    console.log('clipboard.getString() is not supported now.')
+  },
+
+  setString: function (text) {
+    // not support safari
+    if (typeof text === 'string' && text !== '') {
+      const tempInput = element()
+      tempInput.value = text
+
+      tempInput.select()
+      document.execCommand('copy')
```

Comment:
what if there's no such method named `execCommand` in document, there may be a error throw out ? Protection code should be inserted here.